amazon-ecr-credential-helper.spec

```diff
@@ -139,6 +139,8 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Wed 18 Sep 2023 Christopher R. Miller <milrchr@amazon.com> - 0.9.0-1
+- Update to v0.9.0
```

Comment:
may be add some details here?

Reply:
Yeah, we don't generally do that elsewhere in the distribution, but it's been consistently done for this package. I've just copy-pasted the items from https://github.com/awslabs/amazon-ecr-credential-helper/releases which appears to have been the convention previously followed.